components/rename_channel_modal/rename_channel_modal.test.tsx

```diff
@@ -3,19 +3,53 @@
 
 import React from 'react';
 import {RequestStatus} from 'mattermost-redux/constants';
+import {Channel} from 'mattermost-redux/types/channels';
+import {Team} from 'mattermost-redux/types/teams';
 
 import {shallowWithIntl} from 'tests/helpers/intl-test-helper';
-import RenameChannelModal from 'components/rename_channel_modal/rename_channel_modal.jsx';
+
+import RenameChannelModal, {RenameChannelModal as RenameChannelModalClass} from './rename_channel_modal';
 
 describe('components/RenameChannelModal', () => {
-    const channel = {
+    const channel: Channel = {
```

Comment:
For future reference, we have a full `Channel` object mocked out in `utils/test_helper` if you wanted to use that.